static/vendor/socket.io.js

```diff
@@ -0,0 +1,9 @@
+/*!
```

Comment:
I downloaded this here, because my computer went offline at the cafe, and I thought I wasn't going to be able to work on it due to it not loading.